src/Redux.elm

```diff
@@ -24,25 +25,20 @@ view model =
 program :
     { init : ( model, Cmd msg )
     , update : msg -> model -> ( model, Cmd msg )
+    , encode : model -> Json.Value
```

Comment:
hmmm... is the json encoding really necessary? If so, I would prefere if the dispatch would decode it.